<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaneInvoiceSettings</class>
 <widget class="QWidget" name="PaneInvoiceSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1520</width>
    <height>1233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options de facturation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QTableView" name="tableViewParams">
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>63</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTableView" name="tableViewInvoicing">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>300</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Informations légales en bas de facture</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPlainTextEdit" name="textEditBottomLegal"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Entreprise</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPlainTextEdit" name="textEditCompany"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Loi indiquée en bas de facture</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPlainTextEdit" name="textEditBottomLaw"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonAddDate">
       <property name="text">
        <string>Add date</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRemoveDate">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
